huaweicloud/services/dew/resource_huaweicloud_csms_secret.go

```diff
@@ -265,40 +265,40 @@ func resourceCsmsSecretUpdate(ctx context.Context, d *schema.ResourceData, meta
 	if d.HasChange("tags") {
 		err = utils.UpdateResourceTags(client, d, serviceType, id)
 		if err != nil {
-			e := fmtp.Errorf("failed to update CSMS secret tags: %s", err)
+			e := fmt.Errorf("failed to update CSMS secret tags: %s", err)
 			mErr = multierror.Append(mErr, e)
 		}
 	}
 
 	if mErr.ErrorOrNil() != nil {
-		return fmtp.DiagErrorf("failed to update CSMS secret: %s", mErr)
+		return diag.Errorf("failed to update CSMS secret: %s", mErr)
 	}
 	return resourceCsmsSecretRead(ctx, d, meta)
 }
 
 func resourceCsmsSecretDelete(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	config := meta.(*config.Config)
-	region := config.GetRegion(d)
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
 	// The endpoint of CSMS is the endpoint of KMS.
-	client, err := config.KmsV1Client(region)
+	client, err := cfg.KmsV1Client(region)
 	if err != nil {
-		return fmtp.DiagErrorf("failed to create HuaweiCloud CSMS(KMS) client: %s", err)
+		return diag.Errorf("failed to create CSMS(KMS) client: %s", err)
 	}
 
 	name := d.Get("name").(string)
 	err = secrets.Delete(client, name)
 	if err != nil {
-		return fmtp.DiagErrorf("failed to delete CSMS secret: %s", err)
+		return diag.Errorf("failed to delete CSMS secret: %s", err)
 	}
 	d.SetId("")
 	return nil
 }
 
-func resourceCsmsSecretImport(ctx context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData,
+func resourceCsmsSecretImport(_ context.Context, d *schema.ResourceData, _ interface{}) ([]*schema.ResourceData,
 	error) {
 	id, name := parseID(d.Id())
 	if id == "" {
-		err := fmtp.Errorf("Invalid format specified for the ID of CSMS secret. " +
+		err := fmt.Errorf("Invalid format specified for the ID of CSMS secret. " +
```

Comment:
error message should start with lower case, please check other messages
```suggestion
		err := fmt.Errorf("invalid format specified for the ID of CSMS secret. " +
```

Reply:
fixed